dayrepo/snippets/views.py

```diff
@@ -47,19 +59,98 @@ def get(self, request):
         return render(
             request,
             "snippet_post.html",
-            {"form": SnippetForm, "form_trouble": DutiesTroubleForm, "form_process":ProcessForm},
+            {
+                "form": SnippetForm,
+                "form_trouble": DutiesTroubleForm,
+                "form_process": ProcessForm,
+            },
         )
 
     # 投稿機能
     def post(self, request):
-        # formに書いた内容を格納する
-        form = SnippetForm(request.POST)
-        form_trouble = DutiesTroubleForm(request.POST)
-        # 保存
-        form.save()
-        form_trouble.save()
+        print(request.POST)
+        req = request.POST
+
+        # Snippets form
+        account = Account(
+            id = req.get("account_id"),
+        )
+        car = Car(
+            id = req.get("car_id"),
+        )
+        gasoline = req.get("gasoline_amount")
+        if gasoline == "":
+            gasoline = 0.0
+
+        oil = req.get("oil")
+        if oil == "":
+            oil = 0.0
```

Comment:
```suggestion
        oil = req.get("oil")
        if oil == "":
            oil = 0.0

```

細かいですが、改行の位置を修正したいですmm